interfaces/mount/spec_test.go

```diff
@@ -91,3 +92,27 @@ func (s *specSuite) TestSpecificationIface(c *C) {
 		{Name: "connected-plug"}, {Name: "connected-slot"},
 		{Name: "permanent-plug"}, {Name: "permanent-slot"}})
 }
+
+const snapWithLayout = `
+name: vanguard
+layout:
+  /usr:
+    bind: $SNAP/usr
+  /mytmp:
+    type: tmpfs
+    user: nobody
+    group: nogroup
```

Comment:
```
 /mytmp:
    type: tmpfs
    user: nobody
    group: nogroup
```

Why are we allowing people to specify nobody/nogroup (or users in general). This user and group is in place for unmapped NFS uid and gids and using it outside of this context is a mistake (though a common one). What is the purpose of this?

Reply:
This is an example we've been working with at the London sprint. I just used it as it was familiar to various participants. It was also used on the forum https://forum.snapcraft.io/t/layouts-re-mapping-snap-directories/1471 in July last year. Currently we only allow "root" and "nobody" and no other names. I can easily change that as this feature is not live yet.

Reply:
Please see: https://forum.snapcraft.io/t/layouts-re-mapping-snap-directories/1471/56